src/Components/Wizard.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { css, withStyles } from "../withStyles";
+
+import FlexContainer from "../Containers/FlexContainer";
+import Modal from "./Modal";
+import Heading from "../Elements/Heading";
+import Paragraph from "../Elements/Paragraph";
+import Button from "../Elements/Button";
+import Input from "../Elements/Input";
+import InputWithIcon from "../Elements/InputWithIcon";
+import Icon from "../Elements/Icon";
+
+class Wizard extends React.Component {
+  constructor({ styles, ...props }) {
+    super(props);
+    this.state = {
+      currentPage: 0
+    };
+
+    this.nextPage = this.nextPage.bind(this);
+    this.previousPage = this.previousPage.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  nextPage(e) {
+    e.preventDefault();
+    this.state.currentPage < this.props.children.length - 1
+      ? this.setState({ currentPage: this.state.currentPage + 1 })
+      : this.setState({ currentPage: 0 });
+  }
+
+  previousPage(e) {
+    e.preventDefault();
+    this.state.currentPage > 0
+      ? this.setState({ currentPage: this.state.currentPage - 1 })
+      : this.setState({ currentPage: 0 });
+  }
+
+  handleSubmit(e) {
+    /** Handle the finished settings */
+    console.log(e.target);
+  }
+
+  dummyFunction(e) {
+    /** This is only used to prevent the button for breaking during storybook */
+    e.preventDefault();
+  }
+
+  /** TODO: Set up redux with each input from each step so we can handle all the settings at once */
```

Comment:
Focus on just being able to pass the data from the Wizard up to "something" else.